This adds a guidance how to request access to GCP (#2336)
alphagov/govuk-developer-docs: source/manual/set-up-gcp-account.html.md

---
owner_slack: "#govuk-developers"
title: Set up your GCP account
section: AWS accounts
layout: manual_layout
parent: "/manual.html"
last_reviewed_on: 2019-02-20
review_in: 6 months
---

> [Static mirrors of GOV.UK](/manual/fall-back-to-mirror.html) are hosted in storage buckets within AWS and Google Cloud Platform (GCP). You may need login to GCP to [remove an asset](/manual/howto-manually-remove-assets.html) or [emargency publish content using static mirror](manual/fall-back-to-mirror.html#emergency-publishing-using-the-static-mirror).

Reviewer: fix for typo:

```suggestion
[Static mirrors of GOV.UK](/manual/fall-back-to-mirror.html) are hosted in storage buckets within AWS and Google Cloud Platform (GCP). You may need login to GCP to [remove an asset](/manual/howto-manually-remove-assets.html) or [emergency publish content using static mirror](manual/fall-back-to-mirror.html#emergency-publishing-using-the-static-mirror).
```

## 1. Request GCP access

> Permission to Google Cloud Provider (GCP) is managed by RE GOV.UK team. Access to GCP is granted when [permanent Production access](manual/rules-for-getting-production-access.html) is approved and merged to 

Reviewer: ```suggestion
Permission to Google Cloud Platform (GCP) is managed by RE GOV.UK team. Access to GCP is granted when [permanent Production access](manual/rules-for-getting-production-access.html) is approved and merged to 
```

[GOV.UK user reviewer](https://github.com/alphagov/govuk-user-reviewer) repository. 

> To request your GCP access login to [GOV.UK Support Portal](manual/zendesk.html) and click on `+Add` to open a new ticket. Please assign your ticket to `3rd Line--GDS Reliability Engineering` team and include a URL for PR where permanent GOV.UK access granted within [GOV.UK user reviewer](https://github.com/alphagov/govuk-user-reviewer) repository.

Reviewer 2: ```suggestion
To request GCP access, [raise a Zendesk ticket with Reliability Engineering](https://docs.publishing.service.gov.uk/manual/raising-issues-with-reliability-engineering.html#raising-a-zendesk-ticket-with-reliability-engineering). Include the user reviewer PR in the ticket body.
```

> To request your GCP access login to [GOV.UK Support Portal](manual/zendesk.html) and click on `+Add` to open a new ticket. Please assign your ticket to `3rd Line--GDS Reliability Engineering` team and include a URL for PR where permanent GOV.UK access granted within [GOV.UK user reviewer](https://github.com/alphagov/govuk-user-reviewer) repository.

Reviewer: Suggested edit for clarity.
```suggestion
To request your GCP access, log in to [GOV.UK Zendesk](manual/zendesk.html) and click on `+Add` to open a new ticket. Please include a link to the [govuk-user-reviewer PR](https://github.com/alphagov/govuk-user-reviewer) which granted you permanent access to GOV.UK. Assign your ticket to `3rd Line--GDS Reliability Engineering` team.
```

> To request your GCP access login to [GOV.UK Support Portal](manual/zendesk.html) and click on `+Add` to open a new ticket. Please assign your ticket to `3rd Line--GDS Reliability Engineering` team and include a URL for a PR where permanent GOV.UK access granted within [GOV.UK user reviewer](https://github.com/alphagov/govuk-user-reviewer) repository.

Author: ```suggestion
To request your GCP access login to [GOV.UK Support Portal](manual/zendesk.html) and click on `+Add` to open a new ticket. Please assign your ticket to `3rd Line--GDS Reliability Engineering` team and include a URL for PR where permanent GOV.UK access granted within [GOV.UK user reviewer](https://github.com/alphagov/govuk-user-reviewer) repository.
```

> 👉 [Request an GCP access](https://govuk.zendesk.com/agent/dashboard)

Reviewer 2: ```suggestion
```
